account_move_line_tax_editable/models/account_move_line.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+
+    _inherit = 'account.move.line'
+
+    is_tax_editable = fields.Boolean(
+        string="Is tax data editable?", compute='_compute_is_tax_editable')
+
+    @api.multi
+    @api.depends('move_id.state')
```

Comment:
```is_tax_editable``` is not stored. I added depends decorator then it's easier to know on what depends the condition, but I understand it can be confusing. I will remove it.
In the relation to the related field, in my opinion it's better to have a computed field instead of a related one. If somebody uses this module and wants to add a condition, he will have to inherit the view and rewrite completely the ```attrs``` tag of ```tax_line_id``` and ```tax_ids```.